addOns/fuzz/src/main/javahelp/org/zaproxy/zap/extension/fuzz/resources/help/contents/options.html

```diff
@@ -3,26 +3,43 @@
 <HEAD>
 <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
 <TITLE>
-Options Fuzz screen
+Options Fuzz Screen
 </TITLE>
 </HEAD>
 <BODY>
-<H1>Options Fuzzer screen</H1>
+<H1>Options Fuzzer Screen</H1>
 <p>
 This screen allows you to configure the <a href="concepts.html">fuzzing</a> options:
 
 <H3>Default category</H3>
```

Comment:
These should be updated accordingly to #3135.

Reply:
All the changes have been made accordingly to #3135